testNormalApiCall:java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.yfy.rpc.demo.test.ConsumerFunctionalTest.main(ConsumerFunctionalTest.java:23)
Caused by: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertNotNull(Assert.java:712)
	at org.junit.Assert.assertNotNull(Assert.java:722)
	at com.yfy.rpc.demo.builder.ConsumerBuilder.testNormalApiCall(ConsumerBuilder.java:51)
	... 5 more

testNormalSpringCall:java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.yfy.rpc.demo.test.ConsumerFunctionalTest.main(ConsumerFunctionalTest.java:23)
Caused by: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertNotNull(Assert.java:712)
	at org.junit.Assert.assertNotNull(Assert.java:722)
	at com.yfy.rpc.demo.builder.ConsumerBuilder.testNormalSpringCall(ConsumerBuilder.java:58)
	... 5 more

testTimeoutCall:pass
testCatchException:java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.yfy.rpc.demo.test.ConsumerFunctionalTest.main(ConsumerFunctionalTest.java:23)
Caused by: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.fail(Assert.java:95)
	at com.yfy.rpc.demo.builder.ConsumerBuilder.testCatchException(ConsumerBuilder.java:86)
	... 5 more

testFutureCall:java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.yfy.rpc.demo.test.ConsumerFunctionalTest.main(ConsumerFunctionalTest.java:23)
Caused by: java.lang.RuntimeException: Thread [Thread[main,5,main]] have not set the response future!
	at com.yfy.rpc.async.ResponseFuture.getResponse(ResponseFuture.java:18)
	at com.yfy.rpc.demo.builder.ConsumerBuilder.testFutureCall(ConsumerBuilder.java:104)
	... 5 more

testCallback:java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.yfy.rpc.demo.test.ConsumerFunctionalTest.main(ConsumerFunctionalTest.java:23)
Caused by: java.lang.RuntimeException: The response doesn't come back.
	at com.yfy.rpc.demo.service.RaceServiceListener.getResponse(RaceServiceListener.java:19)
	at com.yfy.rpc.demo.builder.ConsumerBuilder.testCallback(ConsumerBuilder.java:125)
	... 5 more

testRpcContext:java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.yfy.rpc.demo.test.ConsumerFunctionalTest.main(ConsumerFunctionalTest.java:23)
Caused by: java.lang.NullPointerException
	at com.yfy.rpc.demo.builder.ConsumerBuilder.testRpcContext(ConsumerBuilder.java:140)
	... 5 more

testConsumerHook:java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.yfy.rpc.demo.test.ConsumerFunctionalTest.main(ConsumerFunctionalTest.java:23)
Caused by: java.lang.NullPointerException
	at com.yfy.rpc.demo.builder.ConsumerBuilder.testConsumerHook(ConsumerBuilder.java:150)
	... 5 more

